app/controllers/admin/mentors_controller.rb

```diff
@@ -46,6 +46,8 @@ def destroy
   def mentor_params
```

Comment:
Would it make sense to move this to `Policy#permitted_attributes`? Isn't it there already?

Reply:
The only difference I can see is that `UsersController` accepts a `traits_list`, and this one accepts a `trait_ids`, because we're using a checkbox list instead of the token field. We can always add `traits_list`. to the `UserPolicy#permitted_attributes` and use that in the controller.

Reply:
that sounds good to me.